src/Init/Data/String/Basic.lean

```diff
@@ -256,15 +256,50 @@ def atEnd : (@& String) → (@& Pos) → Bool
   | s, p => p.byteIdx ≥ utf8ByteSize s
 
 /--
-Similar to `get` but runtime does not perform bounds check.
+Returns the character at position `p` of a string.
+If `p` is not a valid position, returns `(default : Char)`.
+
+Requires evidence, `h`, that `p` is within bounds
+instead of performing a runtime bounds check as in `get`.
+
+Examples:
+Given `def abc := "abc"` and `def lean := "L∃∀N"`,
+* `abc.get' 0 (by decide) = 'a'`
+* `lean.get' (0 |> lean.next |> lean.next) (by decide) = '∀'`
```

Comment:
May clearer without the defs, and just inlining the strings?

Reply:
and below.

Reply:
I updated the single `"abc"` case. I agree this is more apparent.

I have streamlined the repetitive examples by replacing the "Given, `def...`" line with a let statement in the bullet:
```lean
`let lean := "L∃∀N"; lean.get' (0 |> lean.next |> lean.next) (by decide) = '∀'`
```

It seems off to me to inline the string in the repetitive cases: 
```lean
`"L∃∀N".get' (0 |> "L∃∀N".next |> "L∃∀N".next) (by decide) = '∀'`.
```

However, please let me know if you still prefer the inlining in the repetitive cases, and I will update the examples accordingly.

Reply:
Looks good, thanks.